lib/utils.bash

```diff
@@ -3,8 +3,11 @@
 set -euo pipefail
 
 GH_REPO="https://github.com/opentofu/opentofu"
+TOOL_BIN_NAME="tofu"
 TOOL_NAME="opentofu"
 TOOL_TEST="tofu version"
+SKIP_VERIFY=${ASDF_OPENTOFU_SKIP_VERIFY:-"false"}
+RELEASE_FILE="${ASDF_DOWNLOAD_PATH}/${TOOL_NAME}-${ASDF_INSTALL_VERSION}.zip"
```

Comment:
The `ASDF_DOWNLOAD_PATH` variable is not passed to `list-all` and `latest-stable`; both fail with `ASDF_DOWNLOAD_PATH: unbound variable`.

I suggest trying to move it inside the `install_version` func, that may work.

ex: 
```bash
local release_file="${ASDF_DOWNLOAD_PATH}/${TOOL_NAME}-${ASDF_INSTALL_VERSION}.zip"
```

Reply:
That could work, but we also need that in the download script and I want to avoid declaring it twice at two different places.

I created a `get_release_file` in `utils.bash` instead and I'm calling it in `download_release` and `install_version `.

For that, I had to move the unzip from the `download` to `download_release` in utils. 
Does that sound good to you?